feature-libs/cart/schematics/add-cart/__snapshots__/index_spec.ts.snap

```diff
@@ -44,10 +44,20 @@ import { CmsConfig, I18nConfig, provideConfig } from \\"@spartacus/core\\";
       [CART_BASE_FEATURE]: {
         module: () =>
           import('@spartacus/cart/base').then((m) => m.CartBaseModule),
-      }, [MINI_CART_FEATURE]: {
+      },
+    }
+  }),
+  provideConfig(<CmsConfig>{
```

Comment:
Hmm, no, I think it's just a consequence of some internal refactoring because now we have to provide more granular configurations.
I actually prefer it to be like this, as it allows for easy config shifting.